Sprint-1/implement/max.js

```diff
@@ -1,4 +1,23 @@
 function findMax(elements) {
+  const numericValues = elements.filter(
+    (element) => typeof element === "number" && !isNaN(element)
+  );
+  let maxNumber = numericValues[0];
+  //console.log(numericValues);
+  if (
+    elements.length === 0 ||
+    elements.every((element) => typeof element != "number")
```

Comment:
If `elements` is `["A", "B", "C", "D"]`, what do you expect `numericValues` be after line 4?

The check at line 9 is very similar to the check at line 3. Instead of carrying out another check at line 9,
why not just take advantage of `numericValues`?

Reply:
with the array of ["A","B","C","D"], numericValues will be:[]
At line 9 ,I removed the check statement and insert this: "if numericValues !=0"
Thanks